spec/features/user_signs_out_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+feature 'user signs out' do
+  scenario "an existing user specifies valid email and password" do
+    user = FactoryGirl.create(:user)
+
+    visit root_path
```

Comment:
If you find that you use this sign-in logic a lot, consider extracting it into a helper!